helix-view/src/editor.rs

```diff
@@ -633,6 +633,7 @@ impl Default for Config {
             bufferline: BufferLine::default(),
             indent_guides: IndentGuidesConfig::default(),
             color_modes: false,
+            newline_at_eof: false,
```

Comment:
maybe you could rename it to a shorter name like?
```suggestion
            eof: false,
```

also could you add a doc comment?

Reply:
Thanks for sending your changes, I used them to clean up the code a bit since I'm not all that well familiar with the helix codebase. But I don't think that the option needs to be renamed, it's not that long and `eof` is not really descriptive for what is does.

Reply:
Hmm that's true, `eof` is vague at best. Glad you found what I did useful :grin:

Reply:
@Xalfer You could consider `eof-newline`, where `eof` is the general topic, and `newline` is the detail applied to that topic.